composer.json

```diff
@@ -22,6 +22,7 @@
         "pestphp/pest-plugin": "dev-master",
         "pestphp/pest-plugin-coverage": "dev-master",
         "phpunit/phpunit": "^9.1.4",
+        "psr/container": "^1.0",
```

Comment:
I am not even sure we should implement this interface. Realistically, people will **never** use this.

What do you think?